ls1intum/ArTEMiS
# added localisation for student dashboard, fixed issues from initial pull request

### Checklist

### Motivation and Context
In the initial pull request for the student dashboard some issues where left open

### Description
added localisation for the student dashoard, fixed issues with linting

addressed issues from https://github.com/ls1intum/ArTEMiS/pull/154#issuecomment-467422615

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to the new student course dashboard
3. Check if the localization is working
4. check if issues from https://github.com/ls1intum/ArTEMiS/pull/154#issuecomment-467422615 are resolved

### Screenshots
![image](https://user-images.githubusercontent.com/10998002/53736889-4c1f3280-3e8b-11e9-9f2a-ce1ac6abd046.png)
